src/test/java/com/ft/content/model/ContentTest.java

```diff
@@ -51,7 +51,8 @@ public void setUp() throws Exception {
                 .withCanBeDistributed(Distribution.YES)
                 .withRightsGroup("rights-group")
                 .withMasterSource(new Identifier("source-authority", "id-on-source"))
-                .withAlternativeStandfirsts(new AlternativeStandfirsts("promotionalStandfirst"));
+                .withAlternativeStandfirsts(new AlternativeStandfirsts("promotionalStandfirst"))
+                .withEditorialDesk("/FT/AnEditorialDesk");
```

Comment:
Somewhere you should have an assertion that the built content has this value for editorialDesk.